braindecode/models/deep4.py

```diff
@@ -13,12 +13,17 @@
 
 
 class Deep4Net(nn.Sequential):
-    """
-    Deep ConvNet model from [1]_.
+    """Deep ConvNet model from Schirrmeister et al 2017.
 
-    References
+    Model described in [2]_.
+
+    Parameters
     ----------
+    in_chans : int
+        XXX
```

Comment:
@robintibor you see that many models have no docstrings for parameters. Can you assign someone to fix this?